tests/packages/failingdependency/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,3 @@
+﻿Write-Error "This should fail!"
+$env:ChocolateyExitCode = '15608'
+#throw "This is crap"
```

Comment:
Do we need this line?

Reply:
Of course we do... How else will we know that it's crap?

It's been removed.